src/saturn_engine/worker_manager/config/declarative_job_definition.py

```diff
@@ -0,0 +1,55 @@
+import dataclasses
+import functools
+
+import desert
+import marshmallow
+
+from saturn_engine.core.api import JobDefinition as CoreJobDefinition
+
+from .declarative_base import BaseObject
+from .declarative_pipeline import PipelineSpec
+
+
+@dataclasses.dataclass
+class JobInput:
+    inventory: str
+
+
+@dataclasses.dataclass
+class JobOutput:
+    topic: str
+
+
+@dataclasses.dataclass
+class JobOutputs:
+    default: JobOutput
```

Comment:
Should we make it a list so it could be possible to send output of the same types to different topics?

Reply:
done